sigcap/src/main/java/com/guardanis/sigcap/SignatureInputView.java

```diff
@@ -134,19 +138,41 @@ public SignatureResponse saveSignature() {
         return new SignatureResponse(renderToBitmap());
     }
 
-    public boolean isSignatureInputAvailable(){
+    public boolean isSignatureInputAvailable() {
         return !(signaturePaths.size() < 1 && activeSignaturePaths.size() < 1);
     }
 
     public void setSignatureRequest(SignatureRequest request) {
         this.request = request;
     }
 
+    public SignatureRequest getSignatureRequest() {
+        return request;
+    }
+
     public void setSignatureRenderer(SignatureRenderer renderer) {
         this.renderer = renderer;
     }
 
     public SignatureRenderer getSignatureRenderer() {
         return renderer;
     }
+
+    /**
+     * Get signature paths.
+     * @author Yordan P. Dieguez
+     */
+    public List<List<Path>> getSignaturePaths() {
```

Comment:
I never exposed these List of Paths because their backing array is mutable. If we want to open this up, we should probably be returning cloned instances of these Lists instead

Reply:
It would be a good option :+1: